### DISCUSSION TITLE
Issues raised on Theme change at workbench:color Theme to darkmode

### DISCUSSION THREAD

**Discussion by sowjanyagunupuru** [eclipse-theia/theia#14264] [Q&A]
Hello @msujew 

While changing the color Theme to the darker modes 
The theme shows invisible text 
May I get any suggestions to solve this issue 

In any of  the lighter theme is choosen that's working good
But on choosing darker themes the text is being invisible and borders are none i.e no outlines for the input boxes as shown in the attached figures

I attached samples with one light theme and one dark theme

Can u pls go through it and suggest me the way to resolve this issue

![IMG_20241008_121019](https://github.com/user-attachments/assets/010f5ba9-1bfe-43ca-a413-e593f7d424c2)

![IMG_20241008_121035](https://github.com/user-attachments/assets/c27a8ca1-0e7b-48db-83e8-ae51b1701398)

**Comment by msujew** [comment]
Hey @sowjanyagunupuru,

is this repoducible in the Theia IDE, or only in your specific Theia app? It might be that you included some CSS that messes with Theia's styling.
